code/__DEFINES/guns.dm

```diff
@@ -69,6 +69,11 @@
 #define MANUFACTURER_PGF "the Etherbor Industries emblem"
 #define MANUFACTURER_IMPORT "Lanchester Import Co."
 
+// Misfire chances if the gun's safety is off
+#define GUN_NO_SAFETY_MALFUNCTION_CHANCE_LOW 5
+#define GUN_NO_SAFETY_MALFUNCTION_CHANCE_MEDIUM 40
+#define GUN_NO_SAFETY_MALFUNCTION_CHANCE_HIGH 80
```

Comment:
i feel like these probabilities should be lowered SUBSTANTIALLY. remember that the character is (at least conceptually) "trying" to make sure the gun DOESN'T fire when they trip or stumble. additionally, guns are designed *not* to go off randomly when they're jostled. i think you could make these probabilities a lot lower -- 5, 10, 15, maybe -- and it would still be a memorable mechanic that would keep people on their toes about safeties.

Reply:
Chance for the misfires have been lowered to the values you suggested.